Makefile

```diff
@@ -47,3 +50,9 @@ push: ./manifest-tool $(addprefix push-,$(ALL_ARCH))
 ./manifest-tool:
 	curl -sSL https://github.com/estesp/manifest-tool/releases/download/v0.5.0/manifest-tool-linux-amd64 > manifest-tool
 	chmod +x manifest-tool
+
+vendor: glide.lock
+	glide install -v
+
+test: vendor
+	CGO_ENABLED=0 GOARCH=$(ARCH) go test ./pkg/...
```

Comment:
This is one of the few cases where we actually don't need GOARCH set, as it will always run on the host platform which is automatically "correct" ;)